typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -226,20 +223,11 @@ export class HyperlaneIsmFactory extends HyperlaneApp<FactoryFactories> {
   }
 
   private async deployOpStackIsm(chain: ChainName, config: OpStackIsmConfig) {
-    const recoveredIsm = getDeployedIsms(config.origin, chain, config.type);
```

Comment:
I would prefer to throw an "unimplemented" error rather than just deploy for future error handling to avoid misconfigured opstackhooks <> opstackisms

Reply:
opened https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2898
generally if you wanted to remediate you had to get on this quickly